packages/typespec-ts/test/azureModularIntegration/generated/payload/multipart/src/index.d.ts

```diff
@@ -110,10 +110,7 @@ export declare interface FormDataHttpPartsNonStringFloatOptionalParams extends O
 
 export declare interface FormDataHttpPartsNonStringOperations {
     float: (body: {
-        temperature: {
-            body: number;
-            contentType: "text/plain";
-        };
+        temperature: number;
```

Comment:
This is a un-supported case and the change is introduced by TCGC. The change is expected but not complete. We could further improve the case for advanced mfd support.